universe/archive.go

```diff
@@ -24,6 +25,10 @@ type ArchiveConfig struct {
 	// if the identifier has never been seen before.
 	NewBaseTree func(id Identifier) StorageBackend
 
+	// IgnoreTreeArchive is an archive of all known ignore trees across all
+	// assets.
+	IgnoreTreeArchive IgnoreTreeArchive
```

Comment:
FWIW, the I envisioned things originally is that you almost never serve the ignore tree by itself. It's always accompanied with two additional layers of inclusion proofs: first into the mains supply tree, and then the block header proof into the blockchain.